Source/WebCore/platform/graphics/GlyphBuffer.h

```diff
@@ -112,6 +112,15 @@ class GlyphBuffer {
         m_offsetsInString.append(offsetInString);
     }
 
+    void add(Glyph glyph, const Font& font, GlyphBufferAdvance advance, FloatPoint origin, GlyphBufferStringOffset offsetInString)
```

Comment:
We should factorize this with the already existing add(...) function (default parameter for `origin`)